src/goto-programs/interpreter_evaluate.cpp

```diff
@@ -339,11 +339,19 @@ void interpretert::evaluate(
         evaluate(expr.op0(), dest);
         return;
       }
-      if(expr.has_operands() && !to_integer(expr.op0(), i))
+      else if(
```

Comment:
Sorry, not being changed here, but as I'm trying to understand the code: line 334 suggests there is a _type_ `ID_address_of`?

Reply:
That _surely_ must be in error!